src/App.vue

```diff
@@ -289,6 +298,12 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
       }
     })
 
+    // listen for name request invalid error events
+    this.$root.$on('invalid-name-request', async error => {
```

Comment:
I'm hoping to make as little use of root-level events as possible.
In this case, would it work well to have the modal on the correction view?
Or, do you foresee that this modal will be used by multiple views?

Reply:
Good point, i think long term it should be accessible app wide due to that infamous Profile View where we launch all corrections etc from... but that is a future state we don't know too much about!
I can certainly refactor this to be more local though, should we go that path for now?

Reply:
No, if you think this will be used app-wide then leave it for now. But instead of several custom events, it would be great to have a single custom event with parameters :)

Reply:
Definitely, will refactor this 👍

Reply:
Updated in next commit *